src/common/image.h

```diff
@@ -89,6 +89,18 @@ typedef struct dt_image_raw_parameters_t
   unsigned user_flip : 8; // +8 = 32 bits.
 } dt_image_raw_parameters_t;
 
+typedef enum dt_exif_image_orientation_t
+{
+  EXIF_ORIENTATION_NONE              = 1,
+  EXIF_ORIENTATION_FLIP_HORIZONTALLY = 2,
+  EXIF_ORIENTATION_FLIP_VERTICALLY   = 4,
+  EXIF_ORIENTATION_ROTATE_180_DEG    = 3,
+  EXIF_ORIENTATION_TRANSPOSE         = 5,
+  EXIF_ORIENTATION_ROTATE_CCW_90_DEG = 8,
```

Comment:
I think CCW and CW are inverted.

Reply:
To be precise, I mean CW should be 8 and CCW 6.

Reply:
I didn't changed that and beleive this is correct and not inverted
See curent definition in image.h:
```
/** return the raw orientation, from jpg orientation. */
static inline dt_image_orientation_t dt_image_orientation_to_flip_bits(const int orient)
{
  switch(orient)
  {
    case 1:
      return ORIENTATION_NONE;
    case 2:
      return ORIENTATION_FLIP_HORIZONTALLY;
    case 3:
      return ORIENTATION_ROTATE_180_DEG;
    case 4:
      return ORIENTATION_FLIP_VERTICALLY;
    case 5:
      return ORIENTATION_TRANSPOSE;
    case 6:
      return ORIENTATION_ROTATE_CW_90_DEG;
    case 7:
      return ORIENTATION_TRANSVERSE;
    case 8:
      return ORIENTATION_ROTATE_CCW_90_DEG;
    default:
      return ORIENTATION_NONE;
  }
```

Also, see exif values described here : https://www.daveperrett.com/articles/2012/07/28/exif-orientation-handling-is-a-ghetto/
- when a picture is in case 6, we have to apply a CW rotation to correct it
- when a picture is in case 8, we have to apply a CCW rotation to correct it

Reply:
Ok, I see you're correct. The number is the rotation to be applied. Thanks.